modules/core/shared/src/main/scala/io/circe/Json.scala

```diff
@@ -198,7 +198,7 @@ sealed abstract class Json extends Product with Serializable {
   /**
    * Use implementations provided by case classes.
    */
-  override def hashCode(): Int
+  def hashCode(): Int
```

Comment:
Why remove the `override`? IIRC having it here means if an implementer doesn't provide an implementation they'll get a compiler error, which is a nice check to have.

Reply:
It is redundant at least in Scala 2.13:

```
$ scala
Welcome to Scala 2.13.0 (OpenJDK 64-Bit Server VM, Java 1.8.0_222).
Type in expressions for evaluation. Or try :help.

scala> abstract class A { def hashCode: Int }
defined class A

scala> class B extends A
             ^
       error: class B needs to be abstract. No implementation found in a subclass for deferred declaration
       def hashCode: Int (defined in class A)

scala> case class B() extends A
defined class B
```